manager/api/transport.go

```diff
@@ -124,6 +131,15 @@ func MakeHandler(svc manager.Service) http.Handler {
 	return r
 }
 
+func decodeIdentity(_ context.Context, r *http.Request) (interface{}, error) {
```

Comment:
The same remark applies to requests: utilise headers to pass the token and identity.